packages/server/src/ui/routes/build-view/audit-list/audit-group.jsx

```diff
@@ -58,7 +58,10 @@ const ChangedAuditList = props => {
             onClick={() => props.setSelectedAuditId(audit.id || null)}
           >
             <div className="audit-group__audit-score">
-              <ScoreIcon score={audit.score || 0} />
+              <ScoreIcon
```

Comment:
I think we can revert these too now :)

Reply:
Done. Thanks for noticing it, my bad I missed it with last revert.